rebasehelper/output_tool.py

```diff
@@ -19,216 +19,200 @@
-        for value in six.itervalues(patches):
-            if value:
-                new_max = max([len(os.path.basename(x)) for x in value])
-                if new_max > max_name:
-                    max_name = new_max
-        max_key = max([len(x) for x in six.iterkeys(patches)])
-        for key, value in six.iteritems(patches):
-            if value:
-                for patch in value:
-                    logger_report.info('Patch %s [%s]', os.path.basename(patch).ljust(max_name), key.ljust(max_key))
+    def print_report_file_path(cls):
+        """Print path to the report file"""
+        cls.colored_output(yellow, '%s report' % cls.NAME)
+        print('%s' % os.path.join(cls.app.results_dir, 'report.' + cls.get_extension()))
 
     @classmethod
-    def print_rpms(cls, rpms, version):
-        pkgs = ['srpm', 'rpm']
-        if not rpms.get('srpm', None):
-            return
-        message = '\n{0} (S)RPM packages:'.format(version)
-        cls.print_message_and_separator(message=message, separator='-')
-        for type_rpm in pkgs:
-            srpm = rpms.get(type_rpm, None)
-            if not srpm:
-                continue
-            message = "%s package(s): are in directory %s :"
-            if isinstance(srpm, str):
-                logger_report.info(message, type_rpm.upper(), os.path.dirname(srpm))
-                logger_report.info("- %s", os.path.basename(srpm))
-            else:
-                logger_report.info(message, type_rpm.upper(), os.path.dirname(srpm[0]))
-                for pkg in srpm:
-                    logger_report.info("- %s", os.path.basename(pkg))
+    def print_patches_cli(cls):
+        """Print info about patches"""
+        patch_dict = {
+            'inapplicable': red,
+            'modified': green,
+            'deleted': green}
+
+        for patch_type, color in six.iteritems(patch_dict):
+            cls.print_patches_section_cli(color, patch_type)
 
     @classmethod
-    def print_build_logs(cls, rpms, version):
+    def print_patches_section_cli(cls, color, patch_type):
         """
-        Function is used for printing rpm build logs
+        Print info about one of the patches key section
 
-        :param kwargs: 
-        :return: 
+        :param color: color used for the message printing
+        :param patch_type: string containing key for the patch_dict
         """
-        if rpms.get('logs', None) is None:
+        patches = results_store.get_patches()
+        if not patches:
             return
-        logger_report.info('Available %s logs:', version)
-        for logs in rpms.get('logs', None):
-            logger_report.info('- %s', logs)
+
+        if patch_type in patches:
+            print('\n%s patches:' % patch_type)
+            for patch in patches[patch_type]:
+                cls.colored_output(color, patch)
+
 
     @classmethod
-    def print_summary(cls, path, results=results_store):
+    def colored_output(cls, color, string):
```

Comment:
I'd rather see some generic helper function or method for printing colored or non-colored output. That way it could be used in more places than just in `TextOutputTool`.